data/json/items/tools.json

```diff
@@ -10855,7 +10861,8 @@
             "You play a quick, folksy tune on your fiddle.",
             "As you pull the bow across its strings, it makes an evil hiss.",
             "You play a tune so fierce, it feels like hell's broke loose."
-        ]
+        ],
+        "played_with": ["EYES"]
```

Comment:
The golden fiddle is so precious to you that you simply don't play it (only look at) =).
This item was meant as a rare one, so it probably should have some distinctions (i.e. you can play it without actual playing, with appropriate descriptions).
_That was a joke, but if you say it's dull, then let it be a more regular fiddle._